src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,13 @@
-const numberToReversedDigits = number => {};
+const numberToReversedDigits = number => {
+    //let num = 12345;
+    
+//return String(number).split("").reverse().join("");
+let num = 12345,
+arr = String(num).split("").reverse().map(Number);
```

Comment:
you should have used the `number` argument here, rather than defining the number yourself